tests/datasource/fluent/data_asset/data_connector/test_s3_data_connector.py

```diff
@@ -1091,6 +1090,6 @@ def test_s3_checkpoint_run_using_same_store_prefixes_errors(
         # Happens when an ExpectationIdentifier is being evaluated as ValidationResultIdentifier
         context.build_data_docs()
 
-    with pytest.raises(InvalidKeyError):
```

Comment:
Previously errored on the default eval param action but now that we don't have it, we see the issue in the data docs action (in a slightly different form).